org.robiot.xclicker.yml

```diff
@@ -0,0 +1,78 @@
+app-id: org.robiot.xclicker
```

Comment:
it this app-id valid? It doesn't seem to be website announced on the github project page, and there is no website there.

None of this adhere to the policy.

Reply:
What do you mean with if the app-id is valid?

Reply:
This app-id isn't valid.

Upstream point to `xlicker.xyz` for the website, so the app-id should be `xclicker.xclicker.xyz`

From https://github.com/flathub/flathub/wiki/App-Requirements#application-id
> Each application should have a unique application ID, following the standard reverse-DNS schema. See [the Flatpak documentation](http://docs.flatpak.org/en/latest/conventions.html#application-ids) for more information on this. The **Application ID should be a real URL of a domain that the app author has control over** or where their app is hosted.

(emphasis mine)

Reply:
Oh yeah, I get it now. Should the app-id not be xyz.xclicker.xclicker?